File: Java/src/main/java/com/contidio/sdk/exceptions/AlreadyExistsBackendException.java
package com.contidio.sdk.exceptions;

import org.apache.commons.httpclient.HttpStatus;

import com.contidio.sdk.proto.MetadataProtocol.BackendErrorCode;

@SuppressWarnings("serial")
public class AlreadyExistsBackendException extends BackendException {
	public AlreadyExistsBackendException(BackendErrorCode errorCode, String message) {
		super(errorCode, message, "AlreadyExistsBackendException", HttpStatus.SC_CONFLICT);
	}	
}

File: .NET Core/Exceptions/AlreadyExistsBackendException.cs
using com.contidio.sdk.proto;
using System.Net;

namespace Contidio.Sdk.Exceptions
{
    public class AlreadyExistsBackendException : BackendException
    {
        public AlreadyExistsBackendException(BackendErrorCode errorCode, string message)
            : base(errorCode, message, "AlreadyExistsBackendException", HttpStatusCode.Conflict)
        {
        }
    }
}
